src/doc/rustc/src/lints/levels.md

```diff
@@ -57,6 +58,15 @@ warning: unused variable: `x`
   = note: to avoid this warning, consider using `_x` instead
 ```
 
+## force-warn
```

Comment:
this is in no way an actual blocker, but the lack of symmetry in naming between "forbid" and "force-warn" is kinda sad. Maybe I should open a proposal for "force-deny", heh.

Reply:
force-deny would have a slightly different meaning: it would deny things in dependencies even if cap-lints is set.